dags/bqetl_main_summary.py

```diff
@@ -529,6 +529,20 @@
         depends_on_past=False,
     )
 
+    wait_for_copy_deduplicate_all = ExternalTaskSensor(
+        task_id="wait_for_copy_deduplicate_all",
+        external_dag_id="copy_deduplicate",
+        external_task_id="copy_deduplicate_all",
+        execution_delta=datetime.timedelta(seconds=3600),
+        check_existence=True,
+        mode="reschedule",
+        allowed_states=ALLOWED_STATES,
+        failed_states=FAILED_STATES,
+        pool="DATA_ENG_EXTERNALTASKSENSOR",
+    )
+
+    crashes_daily_v1.set_upstream(wait_for_copy_deduplicate_all)
```

Comment:
In all the other cases the affected tasks had other dependencies on tasks downstream from `copy_deduplicate_all`, but this `telemetry_derived.crashes_daily_v1` task had no other dependencies so it could have ended up running before `copy_deduplicate_all` finished successfully (and wouldn't be re-run if `copy_deduplicate_all` was cleared recursively downstream).

It looks like that did happen at least once, as the `telemetry_derived.crashes_daily_v1` table has zero rows for 2022-12-01, so we should re-run this ETL for that date.

Reply:
I have re-run the `telemetry_derived.crashes_daily_v1` ETL for 2022-12-01 along with all downstream tasks.